spec/features/studios/index_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe "Studio Index Page" do
+  it "displays all the studios name's and locations" do
+    universal_studios = Studio.create!(name: "Universal Studios", location: "Hollywood")
+    disney = Studio.create!(name: "Disney", location: "Hollywood")
+    fox_studios = Studio.create!(name: "Fox Studios", location: "Hollywood")
+
+    tfatf = universal_studios.movies.create!(title: "The Fast and the Furious", creation_year: "2001", genre: "action")    # As a user,
+    shrek = universal_studios.movies.create!(title: "Shrek", creation_year: "2002", genre: "animated")
+  
+    star_wars = disney.movies.create!(title: "A New Hope", creation_year: "1979", genre: "sci-fi")
+    avengers = disney.movies.create!(title: "End Game", creation_year: "2019", genre: "action")
+        
+    ice_age = fox_studios.movies.create!(title: "Ice Age", creation_year: "2006", genre: "animated")
+    avatar = fox_studios.movies.create!(title: "Avatar", creation_year: "2009", genre: "sci-fi")
+    # As a user,
+    # When I visit the studio index page,
+    visit "/studios"
+    # Then I see all of the studios including name and location,
+    expect(page).to have_content("Universal Studios")
```

Comment:
These assertions could also live inside within blocks in order to verify that the location corresponds to the correct studio!